pack_and_terraform.bash

```diff
@@ -106,6 +106,13 @@ if echo "$RUNTF" | grep -iq "^y"; then
     creds_dir=$HOME/.creds/$VPC_NAME
     mkdir -p $creds_dir
 
+	if [ -z "$AWS_S3_ACCESS_KEY" ]; then
+		read -p "Enter your access key to S3 bucket: " AWS_S3_ACCESS_KEY
```

Comment:
Can we adding something about "for saving terraform state" here in the prompt so it is more clear?